src/main/ionic/peers/src/states/user-state.ts

```diff
@@ -29,6 +29,7 @@ export class UserState {
   }
   public updateUser(user) {
     this.appState.setIsLoading(true);
+    user.interests = user.interests.split(',');
```

Comment:
I think it's better to change
public updateUser(user)
to
public updateUser(user: User)
And make sure you pass it a correct user object from the edit profile page

Reply:
Ah, good idea, that fix was too simple...